kiali/README.md

```diff
@@ -0,0 +1,85 @@
+# Install custom Kiali in Kyma
+
+## Overview
+
+The Kyma Kiali component brings limited configuration options in contrast to the upstream [`kiali-operator`](https://github.com/kiali/helm-charts/tree/master/kiali-operator) chart. Modifications might be reset at next upgrade cycle.
+
+An alternative can be a parallel installation of the upstream chart offering all customization options. This tutorial outlines how to achieve such installation in co-existent to the Kyma stack.
```

Comment:
Using `parallel` users might end up installing two kiali instances on their cluster is that acceptable.

Reply:
Two instances do not affect each other. So this should be fine.